Sateltiles/admin_user.py

```diff
@@ -0,0 +1,147 @@
+import wx
+import os
+
+from init import *
+import login as log
+
+class AdminUserPage(wx.Frame):
+    def __init__(self, parent, title):
+        super(AdminUserPage, self).__init__(parent, title=title, size=(600,520), style=wx.MINIMIZE_BOX| wx.SYSTEM_MENU | wx.CAPTION

```

Comment:
dont hard code the value here...read from some settings page if the same value applies at other place too